tests/testenvironment.proj

```diff
@@ -0,0 +1,133 @@
+<Project DefaultTargets="CreateTestEnvFile">
+
+  <!-- COMPlus_* variables that can be specified for a test scenario -->
```

Comment:
It would be nice if you could add a comment describing what this project does. Something like "This project creates a .cmd or .sh file that sets the environment variables corresponding to the specified 'Scenario'."